### ISSUE TITLE
Endomondo not syncing since 2016-02-09 (now:2016-02-14)

### ISSUE THREAD

**Issue by mlidbom** [cpfair/tapiriik#203]
I'm tracking walking workouts at least once daily with the Endomondo android app lately. 
The last one that was synced was on 2016-02-09. Since then none of the workouts in endomondo have turned up in my other accounts.

**Comment by mlidbom** [comment]
And now my latest run from garmin connect (2016-02-14) is not synced either.
(I manually exported the missing walks from endomondo and imported to runkeeper and then they synced with the other services. I've started logging walks with runkeeper instead since that seems to work with tapiriik.)

Tapiriik is getting quite flaky lately. I feel I have to check every workout manually time to make sure that it actually works :(

**Comment by mlidbom** [comment]
Actually it seems to be all new activities on garmin connect. The swim I logged 2016-02-14 was not synced either. I've now manually entered those activities on runkeeper.

**Comment by mlidbom** [comment]
Now the latest two swims (2016-02-16) in garmin connect synced correctly. 
Was something changed? I don't see any commits.

Is this just a "patience young grasshopper" thing? Should I expect that sometimes it might take several days for some activities to sync from some services? Maybe at the moment reading new data from those services fails for some reason such as timeouts?
If that is so **it would be great if the website could show read status from different services in a table**. 
Something like:

> Last sync: 2016-02-10 16:00
> Individual service status:
> Runkeeper: New activities read: 2016-02-10 12:00 (**LAST 2 READ ATTEMPTS FAILED**)
> Garmin Connect: New activities exported( 2016-02-10 16:00) (**OK**)
> 
> NOTE: Intermittent errors such as timeouts will usually correct themselves within a few days. Please wait for at least a week to report bugs about such errors.

**Doing something like this could save you a ton of frustrated users and bug report handling if my guess about what is happening is correct.**

**Comment by ElvisTheKing** [comment]
can confirm, using endomondo and tapiriik doesn't show newer workouts (in my case last was synced on 6 feb)

**Comment by djmason** [comment]
This happens for me - takes a *very* long time to sync from Endomondo into Strava, so long in fact that I've just started doing it manually, after paying for the Tapiriik auto-sync service and not seeing it actually sync.

**Comment by oskarthr** [comment]
I can confirm problem. New activites on Endomondo do not or rearly sync to other apps like Strava or Runkeeper but Strava-> Endomondo sync works. 
Ive tried reconnecting Endomondo again and that works for at least one sync.
Looks like the same problem as https://github.com/cpfair/tapiriik/issues/193#issuecomment-167485356

**Comment by marinadl** [comment]
Can confirm as well. My runs won't sync neither automatically nor manually from Endomondo to Runkeeper, but my strength training activities sync from Runkeeper to Endomondo.

**Comment by Claymm** [comment]
Yeah me too even if I paid for that thing.
I think it might come from Endomondo which is changing their API whenever they want.
Tempted to run the open source service on a box of mine but that won't solve the problem.
Unfortunately I'm not a Python developer :/

**Comment by Claymm** [comment]
Just got my run synced. Not sure what happened. Maybe @cpfair can comment on that.

**Comment by oskarthr** [comment]
Yes, my endomondo workouts last week have been syncing again to other apps like before issue. Maby issue has been fixed?

**Comment by marinadl** [comment]
Same as @oskarthr here. They're now synching normally.

**Comment by RubenAzarja** [comment]
Problems seem to be coming and going. Last Endomondo->Strava sync is from Feb 22. I have a recent Endomondo running activity (Mar 6) that doesn't sync. Dashboard shows ‘Queuing’ all the time.

**Comment by oskarthr** [comment]
@RubenAzarja I can sadly confirm that.
